### ISSUE TITLE
enhancement: Simple Api

### ISSUE THREAD

**Issue by kkuez** [Mnaukal/virtual-loop-pedal#3]
A simple local api would be rly good so one could attach pedals or something else to control the system to the program.

**Comment by Mnaukal** [comment]
Yeah, it could be useful. Unfortunately I don't have time to work on this project now a I won't have it in near future.
